src/backup/BackupComplete.tsx

```diff
@@ -29,7 +31,12 @@ function BackupComplete({ route }: Props) {
   useEffect(() => {
     const timer = setTimeout(() => {
       if (navigatedFromSettings) {
-        navigate(Screens.Settings, { promptConfirmRemovalModal: true })
+        navigate(
+          getFeatureGate(StatsigFeatureGates.USE_TAB_NAVIGATOR)
+            ? Screens.Settings
```

Comment:
Wonder if we have to clear the stack before navigating to the new settings screen with back. Previously we landed back on the drawer and there was no way for the user to go back, but wonder if this opens up the possibility on landing on an inconsistent state in the app.